plugin_runner/plugin_runner.py

```diff
@@ -30,6 +27,10 @@
 from canvas_sdk.utils.stats import get_duration_ms, tags_to_line_protocol
 from logger import log
 
+from .authentication import token_for_plugin
+from .plugin_synchronizer import publish_message
+from .sandbox import Sandbox
```

Comment:
pre-commit hooks moved these imports, but I made them relative imports. This allows me to import the plugin runner from the cli function so I can run it that way, but prevents me from running the plugin the "traditional" way.

`canvas run-plugins "plugin1" "plugin2"` <-- works

`python plugin_runner/plugin_runner.py` <-- doesn't work
```
ImportError: attempted relative import with no known parent package
```

I understand the error, but would like suggestions for remediating.

Reply:
@jamagalhaes do you have any suggestions for how to handle this import?

Reply:
@aduane  I would keep absolute imports because they are usually more readable. In addition to that, PEP 8 explicitly recommends absolute imports https://peps.python.org/pep-0008/#imports.

What you probably want here is to run the module as a script and not a top-level script:

`python -m plugin_runner.plugin_runner`

I just did a commit to this branch, please take a look.

Reply:
That makes sense!